hash/msx1_cart.xml

```diff
@@ -22596,8 +22593,7 @@ legacy FM implementations cannot find it.
 	<software name="leilak">
 		<description>Leila K</description>
 		<year>2007</year>
-		<publisher>&lt;homebrew&gt;</publisher>
-		<info name="developer" value="DVIK &amp; Joyrex productions"/>
+		<publisher>DVIK &amp; Joyrex productions</publisher>
```

Comment:
This is stylised “dvik & joyrex” on their own site (“dvik” isn’t an initialism, it’s from Daniel Vik’s name – D. Vik).